hushline/templates/submission_success.html

```diff
@@ -29,10 +28,6 @@
     Make sure you copy this link before closing this window or navigating to another page.
     Once you do, this link will not be displayed again.
   </p>
-
-  <p>
-    <a class="btn" href="{{ url_for('directory') }}">Continue</a>
```

Comment:
I think we still need a link back to "home" here somehow so the user has a clear exit. Maybe not a button, but at least a link like "Back to the directory," no?

Reply:
I don't know, once the message has been sent I think the only action is exit, but adding a button for that just recreates browser behavior of closing the tab or window.

Reply:
hmm well we'll just leave this change for now an then figure it out later